design/ceph/object/zone.md

```diff
@@ -70,19 +71,29 @@ The order in which these resources are created is not important.
 
 When the storage admin is ready to sync data from another Ceph cluster with multisite set up (primary cluster) to a Rook Ceph cluster (pulling cluster), the pulling cluster will have a newly created in the zone group from the primary cluster.
 
-A [ceph-object-pull-realm](/design/object/ceph-object-pull-realm.md) resource must be created to pull the realm information from the primary cluster to the pulling cluster.
+A [ceph-object-pull-realm](/design/ceph/object/ceph-object-pull-realm.md) resource must be created to pull the realm information from the primary cluster to the pulling cluster.
 
 Once the ceph-object-pull-realm is configured a ceph-object-zone must be created.
 
 After an ceph-object-store is configured to be in this ceph-object-zone, the all Ceph multisite resources will be running and data between the two clusters will start syncing.
 
 ## Deleting and Reconfiguring the Ceph Object Zone
 
-At the moment creating an ceph-object-zone resource only handles Day 1 initial configuration for the zone.
+At the moment creating a CephObjectZone resource does not handle configuration updates for the zone.
 
-Changes made to the resource's configuration or deletion of the resource are not reflected on the Ceph cluster.
+By default when a CephObjectZone is deleted, the pools supporting the zone are not deleted from the Ceph cluster.
 
-To be clear, when the ceph-object-zone resource is deleted or modified, the zone is not deleted from the Ceph cluster. Zone deletion must be done through the toolbox.
+If `preservePoolsOnDelete` is set to false, then the CephObjectZone will be removed only if all CephObjectStores that reference this zone are deleted first.
```

Comment:
This is not the behavior we agreed upon in our last meeting. Pool deletion should be independent of zone deletion.

```suggestion
If `preservePoolsOnDelete` is set to false, the pools supporting the zone are deleted from the Ceph cluster. <!-- and this should be paired with the above statement -->

A CephObjectZone will be removed only if all CephObjectStores that reference the zone are deleted first.
```